src/components/MovieCard/MovieCard/MovieCard.js

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import CardContent from "@mui/material/CardContent";
+import CardMedia from "@mui/material/CardMedia";
+import Typography from "@mui/material/Typography";
+import { CardActionArea, CardActions } from "@mui/material";
+import StyledCardButton from "../StyledCardButton/StyledCardButton";
+import LaunchIcon from "@mui/icons-material/Launch";
+import PlayArrowRoundedIcon from "@mui/icons-material/PlayArrowRounded";
+import StarBorderRoundedIcon from "@mui/icons-material/StarBorderRounded";
+import StarRateRoundedIcon from "@mui/icons-material/StarRateRounded";
+import { Box, Link } from "@mui/material";
+import { StyledCard, StyledWishlistBtn } from "./style";
+
+const MovieCard = ({
+  title = "title",
+  rate = "0.0",
+  image = "https://picsum.photos/id/870/200/300?grayscale&blur=2",
+}) => {
+  return (
+    <StyledCard>
+      <CardActionArea>
```

Comment:
add "disableRipple" in props in CardActionArea to stop giving the ripple effect on each click on the card

Reply:
done